### PR TITLE
create new tuning exercise separate from nested resa

### PR DESCRIPTION
**Pull request by lisa-wm**

### CODE REVIEW

**Comment by ludwigbothmann** [comment]
[create new tuning exercise material](https://trello.com/c/8xbFGson/256-create-new-tuning-exercise-material)

File: exercises/tuning/ex_rnw/ex_tuning_knn.Rnw [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  interested in.
+  Modify the HPO procedure such that performance is optimized w.r.t. AUC.
+  
+  \item Visualize the tuning result with a suitable command. 
+  What do you observe regarding the impact of different hyperparameters on 
+  predictive performance?
+  What are limits of such a form of analysis?
+  
+  \item After analyzing the tuning results, you notice that changes in $k$ are 
+  more more influential for smaller neighborhoods.
```
Comment (ludwigbothmann) [inline exercises/tuning/ex_rnw/ex_tuning_knn.Rnw:37]: remove one "more"

File: exercises/tuning/ex_rnw/sol_tuning_knn.Rnw [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    If $k$ is chosen too small, we may encounter overfitting.
+    Conversely, larger neighborhoods produce a more global model with larger 
+    parts of the input space receiving the same prediction.
+    Setting $k$ too large may result in underfitting.
+    \item Kernel: corresponds to the choice of similarity metric. 
+    It determines the importance weights in the $k$-neighborhood.
+    
+    \includegraphics[width=0.5\textwidth]{figure/kernels.png}
+    
+    \footnotesize{\url{https://en.wikipedia.org/wiki/Kernel_%28statistics%29#/media/File:Kernels.svg}}
```
Comment (ludwigbothmann) [inline exercises/tuning/ex_rnw/sol_tuning_knn.Rnw:20]: Footnotesize does not end after the caption here..

**Review by ludwigbothmann — CHANGES_REQUESTED** [review]
Just 2 mini changes. But there are conflicts that prevent from merging right now. Please resolve those and upload the sheet to moodle (independently from resolving the conflicts..) thx @lisa-wm !

**Comment by lisa-wm** [comment]
> Just 2 mini changes. But there are conflicts that prevent from merging right now. Please resolve those and upload the sheet to moodle (independently from resolving the conflicts..) thx @lisa-wm !

conflicts solved 🙏🏻

**Review by ludwigbothmann — APPROVED** [review]

**Pull request merged**